rho/host_discovery.py

```diff
@@ -135,16 +133,19 @@ def create_ping_inventory(vault, vault_pass, profile_ranges, profile_port,
     # verbosity can break our parsing of Ansible's output. This is
     # a temporary fix - a better solution would be less-fragile
     # output parsing.
-    with stopit.ThreadingTimeout(30 * 60) as timeout_mgr:
+    try:
         ansible_utils.run_with_vault(
             cmd_string, vault_pass,
             log_path=PING_LOG_PATH,
             env=my_env,
             log_to_stdout=process_discovery_scan,
             log_to_stdout_env=log_env,
-            ansible_verbosity=0, error_on_failure=False)
-
-    if timeout_mgr.state == timeout_mgr.TIMED_OUT:
+            ansible_verbosity=0,
+            timeout=30 * 60,
+            error_on_failure=False)
+    except ansible_utils.AnsibleTimeoutException:
+        # If the discovery scan times out, we'll just parse whatever
+        # information we have in the log file.
         log.warning('Host discovery timed out before completion.')
```

Comment:
We should communicate this to the user. Can't remember if a log.warning will go to the console.

Maybe something like:
`'Host discovery timed out. Gathering available host information to proceed with scan.'`

Reply:
A log warning goes to the console even with no verbosity flags, or at least it did for me just now.

I updated the message to what you wrote. Since you already approved, I will merge.